src/elements/common/sub-header/ViewModeChangeButton.scss

```diff
@@ -1,4 +1,4 @@
-.bdl-ViewModeChangeButton {
+.btn.bdl-ViewModeChangeButton {
     padding: 5px 6px;
```

Comment:
conflicting with this: https://github.com/box/box-ui-elements/blob/e5aa7e5adc5e16fb91dad753f4c467dba75ec561/src/styles/common/_buttons.scss#L17